packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -563,6 +564,7 @@ class PageView extends StatefulWidget {
     this.onPageChanged,
     List<Widget> children = const <Widget>[],
     this.dragStartBehavior = DragStartBehavior.start,
+    this.cacheExtent = 0,
```

Comment:
cacheExtent is a bit misleading. It is not unit of pixel but of page. Maybe cachePageExtent or cachePage?

Reply:
done (cachePageExtent)

Reply:
Not sure what I can do about having forked from master when the build is broken. Any guidance is appreciated on how to handle this in the future.